src/ServiceLayer.Mesh/FileTypes/NbssAppointmentEvents/Models/FileTrailerRecordMap.cs

```diff
@@ -0,0 +1,15 @@
+using CsvHelper.Configuration;
+
+namespace ServiceLayer.Mesh.FileTypes.NbssAppointmentEvents.Models;
+
+public sealed class FileTrailerRecordMap : ClassMap<FileTrailerRecord>
```

Comment:
move this into the `ServiceLayer.Mesh.FileTypes.NbssAppointmentEvents` namespace

Maybe as a nested class within `FileParser` if that's acceptable to CsvParser